webapp/tests/karma/unit/services/search.js

```diff
@@ -67,10 +67,15 @@ describe('Search service', function() {
         .then(function(actual) {
           chai.expect(actual).to.deep.equal(expected);
           setTimeout(done); // defer execution to give the second query time to execute
+        })
+        .catch((err) => {
+          window.__karma__.error(err);
         });
       service('reports', {})
         .then(function() {
           throw new Error('the second promise should be ignored');
+        })
+        .catch(() => {
```

Comment:
We should probably handle this somehow. Probably something like...

```suggestion
        .catch(err => done(err));
```